### PR TITLE
Added Feature: choose shown streetnames from zoomlevel abstraction.

### PR DESCRIPTION
**Pull request by JoshCoder42**
Ready to merge

### CODE REVIEW

File: packages/OpenSqueakMap.package/OSMRenderObject.class/instance/showStreetName..st [hunk 1]
```diff
@@ -1,8 +1,10 @@
 morphic
-showStreetName
-	| streetseparator |
-	streetseparator := (self tags at: #highway ifAbsent: []).
-	streetseparator ifNotNil: 
+showStreetName: aSet
+	| streetType |
+	"#highway is key for different street types"
+	streetType := (self tags at: #highway ifAbsent: ['']).
+	Transcript show: aSet.
```
Comment (birneamstiel) [inline packages/OpenSqueakMap.package/OSMRenderObject.class/instance/showStreetName..st]: please remove transcript outputs :-)

File: packages/OpenSqueakMap.package/OSMChunk.class/instance/selectStreetsToShow..st [hunk 2]
```diff
@@ -0,0 +1,6 @@
+helper
+selectStreetsToShow: aMorph
+	
+	self zoomlevelAbstraction at: self viewport zoomLevel ifPresent: [:aSet | Transcript show: aSet. aMorph showStreetName: aSet].
```
Comment (birneamstiel) [inline packages/OpenSqueakMap.package/OSMChunk.class/instance/selectStreetsToShow..st]: one more transcript show:...

**Review by bakoe — CHANGES_REQUESTED** [review]
Please remove the two Transcript calls, especially from `selectStreetsToShow` -- makes the method much more readable and increases overall performance (Transcript calls are quit slow).

**Review by bakoe — APPROVED** [review]

**Review by birneamstiel — APPROVED** [review]

**Comment by birneamstiel** [comment]
@JoshCoder42: Nach dem mergen des `dev`-Branches funktioniert das nicht mehr so richtig... Schaust du dir das nochmal an?

**Pull request merged**